index.html

```diff
@@ -56,10 +56,24 @@
             </div>
 
             <div class="section-sub s1">
-
+                <img class="img new-img" src="./assets/newarrival.jpg">
+                <div class="new-arrivals">
+                    <div>NEW ARRIVALS</div>
+                    <div>
+                        <h4 class="bold">Winter Collection</h4>
+                        <span>Check out our best winter collection and stay warm in fashion. </span>
+                    </div>
+                </div>
             </div>
             <div class="section-sub s2">
-
+                <img class="img new-img" src="./assets/newarrival.jpg">
+                <div class="new-arrivals">
```

Comment:
just a suggestion, maybe use loading attribute with img tags because as of now there are less images on the page but when it increase ,it will start impacting performance.